src/materialized/src/http/static/js/memory.jsx

```diff
@@ -229,8 +229,8 @@ function View(props) {
       setOpers(opers);
 
       const chan_table = await query(`
-        SELECT DISTINCT
```

Comment:
Only now do I notice that there's a DISTINCT in there. I guess it speaks to very equal-weight balancing that we've not been seeing many duplicates from that original query? Or maybe the JS just throws data away...

Reply:
Update: yep, the JS was just [throwing away](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) the duplicate edge information